observation_portal/accounts/management/commands/create_user.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management.base import BaseCommand
```

Comment:
I made all these changes except the env variable one. I'm not sure how that will help I guess. In general if you can run the management command, you could run shell_plus and have full access to any user info anyway. If it's to avoid putting it in the docker-compose file, then you would still need it in there in the env variable right? Maybe I'm not understanding it. 

Also I left the default api token because it's used in the site software testing stack and I'm not sure how to mess with that. It can be set to empty string to trigger a token getting generated for you though.

Reply:
I was thinking about the environment variable for when running on the command line... yeah you're totally right, anyone running that command would be able to get into the db if they wanted anyway.

Can you add a note to the help text of the token saying that?